src/tests/ftest/util/command_utils.py

```diff
@@ -210,7 +211,14 @@ def stop(self):
                     self._command, str(state))
                 self._process.send_signal(signal_to_send)
                 if signal_list:
-                    time.sleep(5)
+                    start = time.time()
+                    try:
+                        self._process.wait(timeout=5)
+                        elapsed = time.time() - start
+                        print('Waited {:.2f}, saved {:.2f}'.format(elapsed,
```

Comment:
It would be better to use `self.log.info()` or `self.log.debug()` here as it will log who is logging.